app/src/main/java/com/duckduckgo/app/settings/SettingsViewModel.kt

```diff
@@ -53,8 +55,6 @@ class SettingsViewModel @Inject constructor(
     val command: MutableLiveData<Command> = MutableLiveData()
```

Comment:
Maybe this should be `SingleLiveEvent<Command>` instead? would that remove the need for `command.value = null` being used?

Reply:
Ohhh that a great idea!